### PR TITLE
add clarifications to diff Orgs

### PR DESCRIPTION
**Pull request by shelbeeee**

### CODE REVIEW

File: docs/community/01_ecosystem.md [hunk 1]
```diff
@@ -5,13 +5,13 @@ title: Ecosystem
 
 # How We Work
 
-Radworks is composed of **Orgs**. Orgs represent different groups of contributors that are funded from its treasury. To obtain funding, Orgs must pass annual budget proposals through Radworks governance. Orgs serve the Radworks purpose by building technologies for developers that cultivate internet freedom. 
+Radworks is composed of **Orgs**. Orgs represent different groups of contributors that are funded from its treasury. Orgs serve the Radworks purpose by building technologies for developers that cultivate internet freedom. To obtain funding, Orgs must pass annual budget proposals through Radworks governance.
```
Comment (abbey-titcomb) [inline docs/community/01_ecosystem.md:8]: Link to governance manual or governance section in docs?

File: docs/community/01_ecosystem.md [hunk 2]
```diff
@@ -5,37 +5,38 @@ title: Ecosystem
...
 
-Currently, Radworks funds four separate Orgs:
-
+Currently, Radworks funds four separate Orgs: Radicle Org, Drips Org, Better Internet Foundation Org and the Grants Org. 
 ## Radicle Org
 
 The Radicle Org aims to develop a fully-sovereign code collaboration stack called “Radicle”.
 [Radicle](https://radicle.xyz/) is designed to be a secure, decentralized and powerful alternative to code forges such
-as GitHub and GitLab while preserving user sovereignty and freedom.
+as GitHub and GitLab while preserving user sovereignty and freedom. See the Radicle Org's [community approve strategy for 2024](https://community.radworks.org/t/formal-review-rgp-18-radicle-org-proposal-2024/3421/4).
```
Comment (shelbeeee) [inline docs/community/01_ecosystem.md:19]: What do you think of adding the links to Org proposals like this?

Comment (abbey-titcomb) [inline docs/community/01_ecosystem.md:19]:
```suggestion
as GitHub and GitLab while preserving user sovereignty and freedom. See the Radicle Org's [2024 budget](https://community.radworks.org/t/formal-review-rgp-18-radicle-org-proposal-2024/3421/4).
```

Comment (abbey-titcomb) [reply docs/community/01_ecosystem.md:19]: It looks good @shelbeeee, but let's just say 2024 budget

File: docs/community/01_ecosystem.md [hunk 3]
```diff
@@ -5,37 +5,38 @@ title: Ecosystem
...
+initiation of Radworks' legal analysis and action. All work done by the Foundation Org for Radworks and its other Orgs is funded from the treasury via an annual Org proposal. The Better Internet Foundation maintains separate funds to cover costs related to its own operations, which are not used to fund work done for Radworks and its Orgs. See the Foundation Org's [community approve strategy for 2024](https://community.radworks.org/t/formal-review-rgp-21-foundation-org-proposal-2024/3420/3).
+
+> The Better Internet Foundation does not control treasury funds, nor can it direct product development (Radicle & Drips).
 
 ## Grants Org
 
 The [Radworks Grants](https://github.com/radicle-dev/radicle-grants) Org finds, funds, and guides research and
 development of any FOSS projects that fall outside the purview of currently funded product/engineering orgs (e.g.
-Radicle, Drips).
+Radicle, Drips). See the Grants Org's [community approve strategy for 2024](https://community.radworks.org/t/formal-review-rgp-20-grants-org-proposal-2024-v2/3464).
```
Comment (abbey-titcomb) [inline docs/community/01_ecosystem.md:40]:
```suggestion
Radicle, Drips). See the Grants Org's [2024 budget](https://community.radworks.org/t/formal-review-rgp-20-grants-org-proposal-2024-v2/3464).
```

File: docs/community/01_ecosystem.md [hunk 4]
```diff
@@ -5,37 +5,38 @@ title: Ecosystem
...
-as GitHub and GitLab while preserving user sovereignty and freedom.
+as GitHub and GitLab while preserving user sovereignty and freedom. See the Radicle Org's [community approve strategy for 2024](https://community.radworks.org/t/formal-review-rgp-18-radicle-org-proposal-2024/3421/4).
 
 ## Drips Org
 
 The Drips Org’s mission is to enable FOSS developers to financially thrive by taking on the problem of [FOSS
 sustainability](https://fosssustainability.com/) / funding public goods. [Drips](https://www.drips.network/) develops
 novel [protocols](https://github.com/radicle-dev/drips-contracts), tools and applications that enable new value flows
-for developers in order to receive funds from their users and supporters.
+for developers in order to receive funds from their users and supporters. See the Drips Org's [community approve strategy for 2024](https://community.radworks.org/t/formal-review-rgp-19-drips-org-proposal-2024/3422/2).
```
Comment (abbey-titcomb) [inline docs/community/01_ecosystem.md:26]:
```suggestion
for developers in order to receive funds from their users and supporters. See the Drips Org's [2024 budget](https://community.radworks.org/t/formal-review-rgp-19-drips-org-proposal-2024/3422/2).
```

**Review by abbey-titcomb — APPROVED** [review]
lgtm

**Pull request merged**